proto/attestation/endorsement.proto

```diff
@@ -30,11 +30,15 @@ message TransparentReleaseEndorsement {
   // https://github.com/project-oak/transparent-release/blob/main/docs/claim-transparency.md#the-claim-format
   bytes endorsement = 1;
 
+  // If non-empty, then the endorsed data is not the actual binary, but this
+  // piece of data, which is usually a serialized proto containing digests.
+  bytes attachment = 2;
+
   // The signature for the endorsement.
-  bytes endorsement_signature = 2;
+  bytes endorsement_signature = 3;
```

Comment:
I don't think we can renumber these fields any more, they are already part of the public API.